staging/src/k8s.io/client-go/dynamic/client_test.go

```diff
@@ -114,10 +117,47 @@ func TestList(t *testing.T) {
 				},
 			},
 		},
+		{
+			resource: "rtest/srtest",
+			name:     "normal_list",
```

Comment:
Name should be unique for each test.

Reply:
`non_namespaced_list`? "normal" doesn't make sense. That could be another PR.

Reply:
Have updated `Name` to be unique for each test.